client/lxd_storage_volumes.go

```diff
@@ -762,23 +762,29 @@ func (r *ProtocolLXD) MoveStoragePoolVolume(pool string, source InstanceServer,
 }
 
 // UpdateStoragePoolVolume updates the volume to match the provided StoragePoolVolume struct.
-func (r *ProtocolLXD) UpdateStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePut, ETag string) error {
+func (r *ProtocolLXD) UpdateStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePut, ETag string) (Operation, error) {
 	if !r.HasExtension("storage") {
```

Comment:
Lets switch to using r.CheckExtension while we are at it here.

Reply:
Would it be acceptable if I replace all r.HasExtension checks with r.CheckExtension (where possible) in a separate PR? Since new LTS is around the corner?

Reply:
yep!